aptos-move/move-examples/mint_nft/step3/sources/nft_step3.move

```diff
@@ -0,0 +1,231 @@
+/// This module is the step 3 of our NFT Move tutorial, building on top of step 2. In this module, we introduce an admin account and
```

Comment:
Can we reuse the resource account introduced in step 2 as the admin account?

Reply:
I think we can use the source account as the admin account, but not the resource account? since we don't have the signer of the resource account